view/zend-developer-tools/toolbar/request.phtml

```diff
@@ -41,6 +40,12 @@ $color = ($this->collector->getStatusCode() < 400)
             <span class="zdt-detail-label">Route</span>
             <span class="zdt-detail-value"><?php echo $this->collector->getRouteName(); ?></span>
         </span>
-        <?php echo $this->ZendDeveloperToolsDetailArray('Templates', $this->collector->getTemplateNames()); ?>
+        <?php foreach($this->collector->getViews() as $value): ?>
```

Comment:
SF dumper will be good for printing config, but in this case... I don't think so.